src/test/scala/com/codemettle/reactivemq/VmBrokerTests.scala

```diff
@@ -109,6 +109,9 @@ class VmBrokerTests(_system: ActorSystem) extends TestKit(_system) with FlatSpec
     override protected def beforeAll(): Unit = {
         setLogging()
 
+        // Needed post-5.15.3 for some reason to serialize Integer.
+        System.setProperty("org.apache.activemq.SERIALIZABLE_PACKAGES", "java.lang")
+
```

Comment:
i wouldn't put this in the library, we really don't want to use java serialization at all

i know the project you're using it in might still have some of that, better to do it there. but shouldn't be enabled for all users

Reply:
didn't notice it was for a test and that we were in fact serializing in this test